### DISCUSSION TITLE
<title>Achievements

### DISCUSSION THREAD

**Discussion by Dmitrykevich** [community/community#45455] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**thx! stithx for very fine stickers in my profile, its very cool! i want to more achievements!** (:

**Comment by SilasRodrigues19** [comment]
Hey @Dmitrykevich 

This link has a complete list of achievements you can get and how to get them.

https://github.com/drknzz/GitHub-Achievements

**Reply by Dmitrykevich** [reply]
Wow! Thx!

**Reply by ettaboyle** [reply]
@Dmitrykevich did that answer your question? Can you close the loop and select `mark as answer` on @SilasRodrigues19's response? Thank you 🙏🏻

**Comment by Faiqlatheef** [comment]
After answering two discussions, You will get Galaxy Brain badge
And Add 2 or more branches and create pull request and pull it You will get one
More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
